### DISCUSSION TITLE
Mintjoomla.com down. Cobalt definitly dead?

### DISCUSSION THREAD

**Discussion by LordHelmchen666** [JoomCoder-com/JoomCCK#192] [General]
It seems Cobalt and Mintjoomla.com dead forever now. 
![Screenshot 2024-03-22 at 15-12-36 mintjoomla com is expired](https://github.com/JoomCoder-com/JoomCCK/assets/43477041/c4f64801-9874-45c1-a820-9b924c993aa8)

**Comment by Giorgi625** [comment]
Oh :( I wanted to get some of my posts from there, I hope Sergey will give a backup of mintjoomla.

**Reply by LordHelmchen666** [reply]
I go there today to copy some of the documentation there, but i am also to late. :(

**Reply by Giorgi625** [reply]
I will text Sergey, if he can share backup to us.

**Comment by andiamine** [comment]
you can use Wayback machine:
https://web.archive.org/web/20230601000000*/http://www.mintjoomla.com/

**Comment by Giorgi625** [comment]
Sergey wrote that he will try to renew domain and also open to access source code (do not know exactly what source code he means but anyway)

**Reply by LordHelmchen666** [reply]
Thanks. So hopefully we will not have two competitors working on the same. 
EDIT: The site is back online

**Reply by Giorgi625** [reply]
There was one Russian who should have worked on new version of Cobalt, Sergey gave him access to everything long time ago, but he did nothing to make it work, so I doubt any other person will try to do it, cause it will takes lots of time and effort.

**Reply by Giorgi625** [reply]
Also before Andi started working on it, I've been searching for someone who would do this job, I asked many but noone was ready to work on it.

**Reply by LordHelmchen666** [reply]
It would be great if they found a way to work together somehow.
And we could support them in some way.

**Reply by Giorgi625** [reply]
I would not be willing to work with that Russian, because I do not think that he is honest person at all. I prefer to wait more time for Andi fixing stuff then being dependant on that guy.
